variables.tf

```
@@ -100,19 +100,19 @@ variable "last_access_time_enabled" {
 variable "blob_delete_retention_policy_days" {
   description = "The number of days that deleted blobs should be retained."
   type        = number
-  default     = 35
+  default     = 7
 }
 
 variable "blob_container_delete_retention_policy_days" {
   description = "The number of days that deleted blob containers should be retained."
   type        = number
-  default     = 35
+  default     = 7
 }
 
 variable "blob_restore_policy_days" {
   description = "The number of days in the past to set the maximum point-in-time restore point for containers. Set value to `0` to disable."
   type        = number
-  default     = 30
+  default     = 6
```

i believe the default is 7 days.
```suggestion
  default     = 7
```

---

It must be lower than the value for soft-delete retention.

By default, Azure sets soft-delete retention to 7 days and point-in-time restore to 6 days 🙂